index.html

```diff
@@ -46,7 +46,7 @@ <h1 class="top-name">ELDON RATZLAFF</h1>
 <section class="midle-section">
     <h2 class="projects">My Recent Works</h2>
 <!-- proyect 1 -->
-<div class="works-cards">
+<div class="works-cards works-cards1">
     <div class="project1"></div>
     <div class="proyect-foot1">
         <h2 class="proyect-foot1-p">Project name goes here</h2>
```

Comment:
- [ ] One of the main objectives of this project is to understand how to parse a Figma design to create a UI. From the project requirement, you should stick to the [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=23%3A9&t=WwXca8i3mkeo5TD6-0) as much as possible. But comparing your design to the Figma design, you can see there are some differences
  - Make sure that there are margins between the header section and the work section
  - make sure the first project details are shown
  - You have a horizontal scroll which shows that your project is not responsive.

### YOUR DESIGN
![Screenshot from 2022-12-11 04-11-49](https://user-images.githubusercontent.com/92303891/206886018-db3d5d5b-7ddf-4655-b4b1-8b1304d7eba5.png)

### FIGMA DESIGN
![Screenshot from 2022-12-11 04-22-15](https://user-images.githubusercontent.com/92303891/206886280-e29d4b6a-5452-4a5e-b7ef-84a9ecac800f.png)